crates/api_models/src/enums.rs

```diff
@@ -95,6 +95,7 @@ pub enum Connector {
     Globalpay,
     Globepay,
     Gocardless,
+    Gpayments,
```

Comment:
Can you please comment this out as this is template code.

Follow any past template code PR and comment out at all the necessary places